### PR TITLE
Fix tablet buttons qt59

### PR DESCRIPTION
**Pull request by vladest**
Test steps
- build interface app using Qt 5.9
- Open Tablet 
- Make sure buttons can be pressed

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/831/HighFidelity-Beta-PR10734-a1475bc65f860d4b15a841665711c703647edfc7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/831/HighFidelity-Beta-PR10734-a1475bc65f860d4b15a841665711c703647edfc7.exe

**Comment by YuppyPlays** [comment]
Hi Vladest, is this ready for QA?

**Comment by sethalves** [comment]
Let's QA this with gustavo's build, but a full check will involve someone building with Qt 5.9.

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

**Fail** As I moved my mouse and clicked on each button, they would not open and highlighted without doing anything else.
image

Please let me know if there is anything more I can do/provide to assist.

**Comment by vladest** [comment]
Added backward compatibility with 5.6. Now Gustavo builds should work as well

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/834/HighFidelity-Beta-PR10734-f34faa3ee7e24c3d2dbe5116fc5d8fb61e53ac21.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/834/HighFidelity-Beta-PR10734-f34faa3ee7e24c3d2dbe5116fc5d8fb61e53ac21.exe

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

I can confirm both in HMD w/ tablet & in Desktop w/ tablet all buttons function normally. **Pass**

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1240,7 +1240,7 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
         setKeyboardFocusOverlay(UNKNOWN_OVERLAY_ID);
     });
 
-    connect(this, &Application::aboutToQuit, [=]() {
+    QObject::connect(this, &Application::aboutToQuit, [=]() {
```
Comment (sethalves) [inline interface/src/Application.cpp]: Can you explain why this change was made?

**Comment by OmegaHeron** [comment]
Building against QT5.9 on Ubuntu 16.04 now -- will advise how that works once tested.

**Comment by OmegaHeron** [comment]
**This likely should get testing on Windows with HMD compiled against QT5.9 to be 100%...**

For what I'm able to test (Linux/Desktop mode only against QT 5.9 and separate compile against QT5.6.3) -- QA Pass.

Ubuntu 16.04 Desktop mode using tablet
Master compiled against QT5.9 -- buttons do not work via mouse or xbox controller.
This PR compiled against QT5.9 -- buttons *do* work via mouse or xbox controller.
This PR compiled against QT5.6.3 -- buttons *do* work via mouse or xbox controller.

File: interface/src/Application.cpp (hunk 1 above)
Comment (vladest) [inline interface/src/Application.cpp]: forgot to cleanup

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/839/HighFidelity-Beta-PR10734-59a77c16bd5072d6c10d00302ef66d95a23577c5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/839/HighFidelity-Beta-PR10734-59a77c16bd5072d6c10d00302ef66d95a23577c5.dmg
